packages/@o3r/pipeline/schematics/ng-add/index.ts

```diff
@@ -16,7 +16,7 @@ function ngAddFn(options: NgAddSchematicsSchema): Rule {
     const ownPackageJson = JSON.parse(fs.readFileSync(packageJsonPath, { encoding: 'utf8' })) as PackageJson & { o3rConfig?: { commitHash?: string } };
     const commitHash = ownPackageJson.o3rConfig?.commitHash;
```

Comment:
Sorry I didn't see it in your previous PR but `o3rConfig` is not aligned with what we decided for telemetry. We set `o3r.config.telemetry`, shouldn't we have `o3r.config.commitHash` to be aligned ?

Reply:
well spotted. I will update in this PR

Reply:
it's actually `config.o3r`, not `o3r.config`. PR updated